.circleci/config.yml

```diff
@@ -361,6 +361,12 @@ jobs:
       - *java_deps
       - *attach_generated_sql
       - *copy_generated_sql
+      - run:
+          name: Copy files from private-generated-sql
+          command: |
+            apt update
+            apt install -y rsync
+            rsync --archive /tmp/workspace/private-generated-sql/sql/ sql/
```

Comment:
I'd suggest moving the `rsync` between line 386 and 387, otherwise the private artifacts will get deployed to stage on every CI run since they are not part of `generated-sql`